lib/bindings.js

```diff
@@ -4,10 +4,34 @@ const bindings = require('../build/Release/canvas.node')
 
 module.exports = bindings
 
+Object.defineProperty(bindings.Canvas.prototype, Symbol.toStringTag, {
+  value: 'HTMLCanvasElement'
+})
+
+Object.defineProperty(bindings.Image.prototype, Symbol.toStringTag, {
+  value: 'HTMLImageElement'
+})
```

Comment:
Sorry for the delay. I can't find this in the spec, but both Firefox and Chrome have these `configurable: true`.

```
> Object.getOwnPropertyDescriptor(Image.prototype, Symbol.toStringTag)
Object { value: "HTMLImageElement", writable: false, enumerable: false, configurable: true }
```

```
> Object.getOwnPropertyDescriptor(Object.getPrototypeOf(document.createElement("canvas")), Symbol.toStringTag)
Object { value: "HTMLCanvasElement", writable: false, enumerable: false, configurable: true }
```

Reply:
Good catch, thanks. I confirmed and applied for the others as well.